api/RTCIceTransport.json

```diff
@@ -616,43 +616,44 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCIceTransport/getSelectedCandidatePair",
           "support": {
             "webview_android": {
-              "version_added": null
+              "version_added": false
             },
             "chrome": {
-              "version_added": null
+              "version_added": false
             },
             "chrome_android": {
-              "version_added": null
+              "version_added": false
             },
             "edge": {
-              "version_added": null
+              "version_added": "13",
+              "notes": "As <code>getNominatedCandidatePair()</code>"
```

Comment:
This note should be replaced by [`alternative_name`](https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#alternative_name).